ext/afform/core/Civi/Api4/Utils/AfformSaveTrait.php

```diff
@@ -70,8 +72,7 @@ protected function writeRecord($item) {
       $isChanged('is_dashlet') ||
       (!empty($meta['is_dashlet']) && $isChanged('title'))
     ) {
-      // FIXME: more targeted reconciliation
-      \CRM_Core_ManagedEntities::singleton()->reconcile();
+      \CRM_Core_ManagedEntities::singleton()->reconcile(E::LONG_NAME);
```

Comment:
@totten this FIXME has been staring me down for a long time. Finally fixed!

Reply:
Woot! That should help performance when editing afforms!